packages/storybook/src/utils/utilities.ts

```diff
@@ -198,3 +204,30 @@ function createStorybookConfig(
   );
   return tmpFolder;
 }
+
+export function showStorybookV5Warning(
+  uiFramework:
+    | '@storybook/angular'
+    | '@storybook/react'
+    | '@storybook/html'
+    | '@storybook/web-components'
+    | '@storybook/vue'
+    | '@storybook/vue3'
+    | '@storybook/svelte'
+    | '@storybook/react-native'
+) {
+  logger.warn(
+    `It looks like you're using Storybook version 5.
+    Please note that starting with Nx version 14, Nx will drop support for Storybook version 5.
```

Comment:
`starting with version 14, Nx will drop support`

No need to mention Nx twice.

Reply:
Ha good point!!